WORKSPACE

```diff
@@ -179,6 +179,14 @@ load(
     "git_repository",
 )
 
+# Dependency seems to be gone
+# TODO fix on bazeldnf side
+go_repository(
++    name = "com_github_xi2_xz",
```

Comment:
I think you need to remove the `+` signs, no?

Reply:
:facepalm: